### PR TITLE
Feature/create data model

### PR DESCRIPTION
**Pull request by Ruthmy**
### Hi there! Ruthmy here. 👋

Tis is the PR for the [Create data model](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/creating_data_model_project_v1_2.md#blog-app---creating-a-data-model) milestone project. The goal of this milestone is to set up the project installing the necessary linters and gems and creating the data model.

### Here is a summary of what has been done:

- [x] Create a new rails app with a meaningful name.
- [x] Set PostgreSQL as the database.
- [x] Run the necessary migrations to create the tables and columns that match the ERD diagram. Consider the following:
  - [x] Foreign keys should be included.
  - [x] Pay attention that in posts table the column holding the foreign key to the users table should be named `author_id`.
  - [x] All columns that are foreign keys should have a corresponding index.
- [x] Follow the [GitFlow as a workflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md) to create and unify the different features of the project.
- [x] Adhere to established [best practices for Ruby programming](https://github.com/microverseinc/curriculum-ruby/blob/main/articles/ruby_best_practices.md), ensuring code quality, readability, and maintainability.
---

### Thank you for taking the time to review this PR ⭐!

---

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,221 @@
...
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a id="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (1GORDON) [inline README.md]: Great job @Ruthmy 🤝

- [ ] Kindly fix the broken issues link to make your readme file more professional.
issues

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @Ruthmy  ✋

Good job so far!

## Highlights!
- Database adapter is using Postgresql ✔️ 
- Added create users migration ✔️
- Added create posts migration ✔️
- Added create comments migration ✔️
- Added create likes migration ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Ruthmy) [reply README.md]: ### Hi @1GORDON 
### Thank you very much for your feedback, I will fix it right away ✨ ✨

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @Ruthmy , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ruthmy** [comment]
### Thank you very much @VICTOR-CHUKWU 🎉🎉🎉

**Pull request merged**